File: zero variance/Zero variance.R
# Zero Variance

#Packages
install.packages("readr")
library(readr)

#Read data into R
Z_dataset <- read.csv(file.choose())
View(Z_dataset)

library(ggplot2)
library(ggthemes)

# Use 'apply' and 'var' functions to 
# check for variance on numerical values
apply(Z_dataset, 2, var)

# Check for the columns having zero variance

which(apply(Z_dataset, 2, var)==0) # ignore the warnings

File: zero variance/Zero variance.py
"Zero Variance"

# Python Libraries (Packages)
import pandas as pd

# Read data into Python
Z_data = pd.read_csv("D:/C DRIVE-SSD DATA backup 15-12-2020/Desktop/360digitmg material/Data Preprocessing/DataSets-Data Pre Processing/DataSets/Z_dataset.csv")
Z_data

Z_data.dtypes


Z_data['square.length'].var()
Z_data['square.breadth'].var()
Z_data['rec.Length'].var()
Z_data['rec.breadth'].var()

